src/main/scala/cromwell/engine/file.txt

```diff
@@ -0,0 +1,1000 @@
+| operations/ENniofmMKhiE8ZnogOjH1kEgw7vetLsXKgpwcm9kdWN0aW9u   |
```

Comment:
I thought for sure you were going to point me to something super obvious elsewhere in the PR and make me feel bad about myself.